### PR TITLE
component created

### PR DESCRIPTION
**Pull request by kzverev**
Requesting code review.

### CODE REVIEW

File: app/client/components/HelloWorld.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+import React from 'react';
+
+export default class HelloWorld extends React.Component {
+
+    static propTypes = {
+        text: React.PropTypes.string,
```
Comment (marcopeg) [inline app/client/components/HelloWorld.js:7]: missing the prop type for "name"

File: app/client/components/HelloWorld.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        };
+
+        var nameStyle = {
+            color: 'tomato',
+            border: '2px solid blue'
+        };
+        return (
+            <div>
+                <div style={ background }>
+                    { text } <br />
```
Comment (marcopeg) [inline app/client/components/HelloWorld.js]: probably you don't need the `<br />` here

File: app/styleguide/HelloWorld.guide.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    render() {
+        return (
+            <SGPage>
+
+                <SGSection title="HelloWorld - without properties">
+                    <HelloWorld />
+                </SGSection>
+
+                <SGSection title="HelloWorld - with value property">
+                    <HelloWorld value="Content for the component" />
```
Comment (marcopeg) [inline app/styleguide/HelloWorld.guide.js]: 1. there is no `value` property in the `HelloWorld` component
2. you should try to use the properties `text` and `name` that you have implemented

**Comment by kzverev** [comment]
new version with fixes available

File: app/client/components/HelloWorld.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    render() {
+        var { text, name } = this.props;
+
+        var background = {
+            backgroundColor: 'lime',
+        };
+
+        var nameStyle = {
+            color: 'tomato',
+            border: '2px solid blue'
```
Comment (marcopeg) [inline app/client/components/HelloWorld.js:25]: you are missing a comma `,` at the end of this line :-)
It's not an error but still an important code quality point.

You can run an automatic checking for those kind of stuff by:

`npm run lint`

on your console

Comment (kzverev) [inline app/client/components/HelloWorld.js:25]: Got it =) Are we going to merge this pull request?

**Comment by marcopeg** [comment]
👍

**Pull request merged**